css/index.css

```diff
@@ -137,8 +165,13 @@ h1 {
   /*======= Create a hover state for the anchor tag in the footer nav to have the following styles:
     color: white;
   */
+  .nav-item:hover{
+    background: white;
+  }
+
   
   footer .footer-content .copyright {
     padding: 20px 0;
+    text: center;
      /*======= center the text here */
   }
```

Comment:
Check your work with the dev tools. 
'text: center' is not a CSS setting.
'text-align: center' is.